src/output-pgsql.cpp

```diff
@@ -198,10 +196,9 @@ int output_pgsql_t::pending_relation(osmid_t id, int exists)
         }
 
         auto const &rel = rels_buffer.get<osmium::Relation>(0);
-        return pgsql_process_relation(rel);
+        pgsql_process_relation(rel);
+        return;
```

Comment:
Any particular reason?

Reply:
The new way makes clear that `pgsql_process_relation` doesn't return anything. If we ever change the function this is in to return something again, this new code will break regardless of whether `pgsql_process_relation` returns something or not, helping us to detect that there is a place here that needs looking at.